articles/how-to-make-an-image-carousel-for-your-website/index.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/how-to-make-an-image-carousel-for-your-website/
+title: How to make an Image Carousel for your website using HTML,CSS and JS
+description: This article takes the reader in a step by step process on how to make an image carousel and make their web pages more interactive and lively to the user hence enhancing their experience.
+author: eugiene-kanillar
+date: 2021-03-15T00:00:00-22:00
+topics: []
+excerpt_seperator: <!--moore-->
+images:
+
+ -url: /engineering-education/how-to-make-an-image-carousel-for-your-website/hero.PNG
+  alt: Sample image of the Carousel
+---
+An Image Carousel is like a container of Images which the user can select through by clicking a buttons which directs him forward or backwards in the collection of images. It enables the website to be more interactive and hence enhance the user experience. The collection of Images can be set to change automatically by use of a timer or manually when the user clicks the displayed buttons.
+<!--moore-->
+# Goal
+To add an image carousell to our website to enhance user experience and make the page more lively.
+
+# Prerequisites
+To follow along you should have some basic Knowledge in HTML,CSS and quite some good understanding of JavaScript especially the use of functions in Javascript.
+You should also have a code editor installed in your laptop. The recommended text editor is [visual studio code](https://visualstudio.microsoft.com/downloads/). Click the link to download
+
+# Procedure
+## Step -1 Creating the project
+Open Visual Studio code and create a new folder. In that folder add three files ending with the .html extension, .css extension and .js extension for the HTML,CSS and Javascript files in which you will type the respective codes.
+
+## Step -2 Write the HTML code
+Dive into the HTML file and write the code as illustrated in the figure text below. The code has illustrations so do not worry.
```

Comment:
Maybe you can give s a brief summary of what the code contains because what I can see is that the illustrations are the comments.
So you can have something like in this step we are going to do this and this...in our code, we will create the slides being used ..